### PR TITLE
[43842] Re-design and small changes to modal dialogues (DS) for coherence- fixing qa issues

### PR DESCRIPTION
**Pull request by bsatarnejad**
Fixing spaces and font-size in 
on-boarding video modal,
attribute help text modal,
save query modal,
response body modal in webhooks.

https://community.openproject.org/work_packages/43842/activity

### CODE REVIEW

File: frontend/src/app/spot/styles/sass/common/container.sass [hunk 1]
```diff
@@ -19,3 +19,6 @@
 
   > .spot-list
     margin: $spot-spacing-1 0
+
+  > .op-uc-container
+    padding-bottom: 0
```
Comment (Kharonus) [inline frontend/src/app/spot/styles/sass/common/container.sass]: I think, we shouldn't put any "legacy" classes into our spot classes. this should go to `container-overrides.sass`

**Review by Kharonus — CHANGES_REQUESTED** [review]
Only a small thing about style placement, rest looks fine!

**Comment by oliverguenther** [comment]
:warning: please rebase onto `release/12.3` and change the target branch

File: modules/webhooks/app/cells/views/response_body.erb [hunk 2]
```diff
@@ -8,14 +8,14 @@
   </a>
   <div class="modal-delivery-element">
     <div class="spot-modal--header">
-     <h1 class="spot-subheader-big_bold"> <%= log_entry.class.human_attribute_name('response_body') %> </h1>
+     <h1 class="spot-subheader-big spot-subheader-big_bold"> <%= log_entry.class.human_attribute_name('response_body') %> </h1>
```
Comment (b12f) [inline modules/webhooks/app/cells/views/response_body.erb]: I think headers are bold by default, so you don't need the extra class

File: modules/webhooks/app/cells/views/response_body.erb [hunk 3]
```diff
@@ -8,14 +8,14 @@
...
     </div>
     <div class="spot-divider"></div>
     <div class="spot-modal--body spot-container">
-      <h2 class="spot-body-big_bold">Headers</h2>
+      <h2 class="spot-body-big spot-body-big_bold">Headers</h2>
```
Comment (b12f) [inline modules/webhooks/app/cells/views/response_body.erb]: `spot-subheader-extra-small` is all you need here and below

**Review by b12f — CHANGES_REQUESTED** [review]
Three small changes, then we're good to merge

**Review by b12f — APPROVED** [review]
LGTM!

**Pull request merged**